cmd/update.go

```
@@ -39,17 +44,46 @@ func updateNode(ctx *cli.Context) error {
 func updateSingleNode(name, branch string, updateConfig bool) error {
 	nodePath := nodePath(name)
 	keyPairPath := path.Join(nodePath, "ssh_keypair")
+	configPath := path.Join(nodePath, "config.json")
 	ip, err := getIp(nodePath)
 	if err != nil {
 		return err
 	}
 
 	// Check if we need to update the node config
 	if updateConfig {
-		data, err := ioutil.ReadFile(path.Join(nodePath, "config.json"))
+		// Read the local config file
+		data, err := ioutil.ReadFile(configPath)
 		if err != nil {
 			return err
 		}
+		var cfg config.Config
+		err = json.Unmarshal(data, &cfg)
+		if err != nil {
+			return err
+		}
+
+		// Read ssh private key from `ssh_keypair` file and decode it into a rsa key
+		keyData, err := ioutil.ReadFile(keyPairPath)
+		if err != nil {
+			return err
+		}
+		block, _ := pem.Decode(keyData)
```

Is this error is ignored on purpose?

---

It's not an error. It's some data we don't need.